python/rtypes/pcc/triggers.py

```diff
@@ -0,0 +1,35 @@
+
+
+class Trigger(object):
+
+    def __init__(self, pcc_type, time, action, priority=None):
+        self.pcc_type = pcc_type
+        self.time = time
+        self.action = action
+        self.priority = priority
+
+    def __call__(self, procedure):
+
+        class TriggerProcedure(object):
+
+            def __init__(self, procedure):
+                self.procedure = procedure
+
+            def __call__(self):
+                return self.procedure
+
+        self.TP = TriggerProcedure(procedure)
+
+        return self.TP.__call__()
+
+    
+class Customer():
+    pass
+
+
+@Trigger(Customer, 'before', 'insert')
```

Comment:
Standard for PCC is using " for strings instead of single quotes. Change and keep consistent.